web/src/styles/base/border-colors.scss

```diff
@@ -23,6 +23,9 @@
   border-right-color: $base !important;
 }
 
+.border-primary-extra-extra-light {
+  border-color: $primary-extra-extra-light !important;
```

Comment:
We haven't discussed this [**Refactor Sass classes to no longer use !important for colors** ticket](https://www.pivotaltracker.com/n/projects/2573455/stories/185089904) yet in our Dev Chores Grooming Session, but do we want to/have to continue using this declaration:`!important`?

Reply:
I think we'll have to address my concern on specificity when reviewing the re-factory ticket, for now I plan to continue to use it.